storage_v2/v1adapter/translator.go

```diff
@@ -24,7 +25,7 @@ func ProtoFromTraces(traces ptrace.Traces) []*model.Batch {
 // to OpenTelemetry traces (ptrace.Traces).
 func ProtoToTraces(batches []*model.Batch) ptrace.Traces {
```

Comment:
can we change the name of the function? E.g. ModelBatchesToTraces. Because in some cases it's also useful to have model.Trace as input/output, not just model.Batch, and the naming does not allow that.

Reply:
or even better V1BatchesToTraces